### DISCUSSION TITLE
Plans for Python 3.13 support?

### DISCUSSION THREAD

**Discussion by musicinmybrain** [encode/starlette#2610] [Q&A] [answered]
Is there a tracking issue or a timeline for supporting Python 3.13? While it is not quite at release candidate status yet, it will soon be the system Python in Fedora Rawhide, and [Starlette is not yet ready](https://bugzilla.redhat.com/show_bug.cgi?id=2290559).

(To reproduce in a virtual environment: `pip install -e .[full]; pip install -r requirements.txt; pip install --update trio; python -m pytest`)

**Comment by adriangb** [comment, marked as answer]
Please file an issue in this repo for any incompatibility you encounter with 3.13. But yes we should add 3.13 in CI soon.
